pre-commit

```diff
@@ -17,8 +17,13 @@
 
 function use_bundle() {
   path_to_bundle=$(command -v bundle)
-  if [ -x "$path_to_bundle" -a -d ".bundle" ] ; then
-    return 0
+  if [ -x "$path_to_bundle" ] && [[ -f Gemfile ]] ; then
```

Comment:
Agreed. I broadly prefer `[[]]` when explicitly working with Bash, but if the rest of the script uses `[]`, consistency is valuable.